<launch>

  <arg name="networks_path" default="/home/ubuntu/jetson-inference/data/networks" />
  <arg name="prototxt_path" default="ped-100/deploy.prototxt" />
  <arg name="model_path" default="ped-100/snapshot_iter_70800.caffemodel" />

  <group ns="camera/rgb">
    <node pkg="image_recognition_jetson" type="detect_net_ros" name="detect_net" output="screen">
      <param name="prototxt_path" value="$(arg networks_path)/$(arg prototxt_path)" />
      <param name="model_path" value="$(arg networks_path)/$(arg model_path)" />
      <remap from="image" to="image_rect_color" />
    </node>
    <node pkg="image_recognition_util" type="publish_image_with_recognitions_overlay" name="overlay">
      <remap from="image" to="image_rect_color" />
    </node>
  </group>
</launch>
